packages/schematics/angular/utility/latest-versions.ts

```diff
@@ -8,10 +8,10 @@
 
 export const latestVersions = {
   // These versions should be kept up to date with latest Angular peer dependencies.
-  Angular: '^6.0.3',
+  Angular: '^6.1.0',
```

Comment:
This change will have to be a `feat()` since it changes the generated project.

Reply:
done.